#! /bin/sh /usr/share/dpatch/dpatch-run
## 19-disable-dance-mpc.dpatch by Raphael Bossek <bossekr@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: This is not the real solution for the probelm. Instead the
## DP: DAnCE projects should require `cidl'.

@DPATCH@

--- ACE_wrappers/TAO/CIAO/CIAO.mwc.orig	2004-10-27 20:18:37.000000000 +0200
+++ ACE_wrappers/TAO/CIAO/CIAO.mwc	2004-10-27 20:18:43.000000000 +0200
@@ -4,5 +4,4 @@
 workspace {
   ciao
   tools
-  DAnCE
 }
--- ACE_wrappers/TAO/CIAO/CIAOTAO.mwc.orig	2004-10-27 20:18:55.000000000 +0200
+++ ACE_wrappers/TAO/CIAO/CIAOTAO.mwc	2004-10-27 20:19:03.000000000 +0200
@@ -11,7 +11,6 @@
   ../orbsvcs/orbsvcs/Svc_Utils.mpc
   ../orbsvcs/orbsvcs/Security.mpc
   ../orbsvcs/orbsvcs/IFRService.mpc
-  DAnCE
   ciao
   tools
 }
